packages/quantic/force-app/main/default/lwc/quanticHeadlessLoader/quanticHeadlessLoader.js

```diff
@@ -35,11 +35,7 @@ const cancelInitialSearch = (engineId) => {
  */
 const executeInitialSearch = (engineId) => {
   window.coveoHeadless[engineId].engine.then((engine) => {
-    engine.dispatch(
-      CoveoHeadless.SearchActions.executeSearch(
-        CoveoHeadless.AnalyticsActions.logInterfaceLoad()
-      )
-    );
+    engine.executeFirstSearch();
```

Comment:
Key reason for this PR, since the [action namespaces are being removed](https://github.com/coveo/ui-kit/pull/877). I could have used the loader functions, but opted to switch to the new engine since `HeadlessEngine` will be removed soon too.